spec/spec_helper.rb

```diff
@@ -32,12 +32,12 @@ def stub_govuk_repos(repo_names)
     )
 end
 
- def stub_github_get_pullrequests(repo_name, branch_name)
+ def stub_github_get_pullrequests(repo_name, branch_name, pull_requests: [])
```

Comment:
Rather than adding a third parameter here, I think it would be better to:

1) Do a default stub in the `stub_github_repo` method (returning `[]`, [like I've done here](https://github.com/alphagov/bulk-changer/pull/9/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R46-R51)), so that you don't need to add lots of `stub_github_get_pullrequests` calls everywhere .
2) then just use `stub_github_get_pullrequests(repo_name, branch_name)` (returning `[{}]`), used only in the tests that need the PR to 'exist'.

Does that make sense?